### DISCUSSION TITLE
500  Internal Server Error Sorry, our servers encountered an unexpected error.

### DISCUSSION THREAD

**Discussion by SIBARO** [ibm-skills-network/author-workbench-knowledgebase#77] [Help]
I have an author account with estratego.ai@gmail.com
Whe i try to log in at https://author.skills.network/   I get:
"500  Internal Server Error Sorry, our servers encountered an unexpected error."

But when I log in with other account have no problem.
The last time I was working on the platform i had a problem at create a Lab.  I think that it is the root of the problem.  I labeled the problematic laboratory as ".... BORRAR" to be deleted.  Maybe deleting this, the problem will solve.

**Comment by SIBARO** [comment]
I acces using a direct link to labs, and have no problem:

https://author.skills.network/labs

**Comment by michaelroudnitski** [comment]
Hi @SIBARO thanks for reporting the problem with details. I was able to reproduce the bug. In this case it seems the cover image for [this guided project](https://author.skills.network/quicklabs/6597) is corrupted and causing the home page to crash. We will release a fix shortly!
